cmd/main.go

```diff
@@ -487,3 +419,38 @@ func registerShutdown(shutdown func()) {
 		shutdown()
 	}()
 }
+
+// formatInstanceTypes accepts a list of instance types details, a number of max results, and an output flag
+// and returns a list of formatted strings representing the passed in intance types with at most maxResults number
+// of results. The format of the strings is determined by the output flag. The number of truncated results
+// is also returned.
+// Accepted output flags: "table", "table-wide", "one-line", "simple", "verbose".
+func formatInstanceTypes(instanceTypes []*instancetypes.Details, maxResults int, outputFlag *string) ([]string, int, error) {
+	if outputFlag == nil {
+		return nil, 0, fmt.Errorf("output flag is nil")
+	}
+	if maxResults < 0 {
```

Comment:
We should add the same check here too :)

Reply:
AH! You're right. Thanks for catching that.